core/src/main/scala/quasar/destination/gbq/GBQFlow.scala

```diff
@@ -101,7 +101,7 @@ final class GBQFlow[F[_]: Concurrent](
 
   def delete(ids: IdBatch): Stream[F, Unit] = idColumn traverse_ { (col: Column[_]) =>
     val strs: Array[String] = ids match {
-      case IdBatch.Strings(values, size) => values.take(size).map(strId => "\"" + strId + "\"")
+      case IdBatch.Strings(values, size) => values.take(size).map(strId => "\"" + strId.replaceAll("\"", "\\\\\"") + "\"")
```

Comment:
The replace sequence has so many backslashes because it's a regex escape. I tested it in the repl:
```scala
scala> "asd\"asd".replaceAll("\"", "\\\\\"")
val res0: String = asd\"asd
```